internals/daemon/daemon.go

```diff
@@ -57,6 +57,16 @@ var (
 	sysGetuid       = sys.Getuid
 )
 
+const (
+	// Whether to include maintenance info (pending restarts) and warnings
+	// with every API response. These are turned off right now because
+	// restarts and warnings aren't used at all in Pebble now, and it's not
+	// worth locking state twice on every request for features we don't use.
+	// We could rip out the code for these features, but this will do for now.
+	includeMaintenance     = false
```

Comment:
My only concern here is that these are const. My affect @thp-canonical or @flotter's plans.